packages/remix/src/utils/serverAdapters/express.ts

```diff
@@ -51,7 +51,13 @@ function wrapExpressRequestHandler(
     }
 
     const url = new URL(request.url);
-    const transaction = startRequestHandlerTransaction(url, request.method, routes, hub, pkg);
+    const transaction = startRequestHandlerTransaction(hub, url, routes, pkg, {
+      headers: {
+        'sentry-trace': (req.headers && isString(req.headers['sentry-trace']) && req.headers['sentry-trace']) || '',
+        baggage: (req.headers && isString(req.headers.baggage) && req.headers.baggage) || '',
+      },
+      method: request.method,
+    });
```

Comment:
Just fyi @Lms24 opened an issue to track the parsing of baggage headers if there are multiple ones on one request: https://github.com/getsentry/sentry-javascript/issues/5672